streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2233,72 +2228,6 @@ public Set<StoreBuilder<?>> stores() {
         }
     }
 
-    @Test
-    public void shouldBindStateWithOldProcessor() {
```

Comment:
This test does not use any store... Not sure about it -- if we follow the test name, the two test I updated above cover this case -- thus removing this one (it's either a duplicate, or it's purpose is unclear...)